docker-compose.yml

```diff
@@ -66,9 +66,9 @@ services:
             MYSQL_PASSWORD: appfortests
 
     cli:
-        image: silintl/php7:7.2
-        volumes:
-            - ${COMPOSER_CACHE_DIR}:/composer
+        image: silintl/php7:7.4
+#        volumes:
```

Comment:
Did you mean to commit this? I've found it to help significantly with composer run time.

Reply:
no I didn't, sorry, I'll undo that